modules/justpremiumBidAdapter.js

```diff
@@ -44,7 +44,7 @@ export const spec = {
       const zone = b.params.zone
       const sizes = payload.sizes
       sizes[zone] = sizes[zone] || []
-      sizes[zone].push.apply(sizes[zone], b.sizes)
+      sizes[zone].push.apply(sizes[zone], b.mediaTypes && b.mediaTypes.banner && b.mediaTypes.banner.sizes && b.mediaTypes.banner.sizes.length && b.mediaTypes.banner.sizes)
```

Comment:
I suspect this is not what you want to do.  Currently if the sizes array is empty you push that empty array `[]` onto the array, but with this change you will push `0` onto the array because `[] && [].length && []` evaluates to zero `0` (which is the value of `[].length`).

Reply:
Yup, thanks for catching it! :)